src/index.ts

```diff
@@ -57,7 +57,7 @@ export {default as Button, ButtonProps, ButtonSize, ButtonAnimationDirection} fr
 export {default as Card, CardProps, CardSectionProps, CardSelectionOptions} from './components/card';
 export {default as Carousel, CarouselProps, PageControlPosition} from './components/carousel';
 export {default as Checkbox, CheckboxProps} from './components/checkbox';
-export {default as ChipsInput, ChipsInputProps, ChipsInputChipProps} from './components/chipsInput';
+export {default as ChipsInput, ChipsInputProps, ChipsInputChipProps} from './incubator/ChipsInput';
```

Comment:
You can actually move the component to the right path, under src/components, Where the old component were. 
Same goes for the example screen.